### ISSUE TITLE
v4.0.1 runtime error "tween.js"

### ISSUE THREAD

**Issue by whyboris** [rintoj/ngx-virtual-scroller#416]
```
    ERROR in node_modules/ngx-virtual-scroller/virtual-scroller.d.ts:1:23 
        - error TS2688: Cannot find type definition file for 'tween.js'.

    1 /// <reference types="tween.js" />
```
The previous version worked fine with _Angular 9_ ... 
I deleted _node_modules_ and re-installed in my project, but the error still persists 😕 

I manually tried adding the types with `npm install --save-dev @types/tween.js` into [my project](https://github.com/whyboris/Video-Hub-App)

Now the error is:
```
ERROR in ./node_modules/ngx-virtual-scroller/__ivy_ngcc__/fesm5/ngx-virtual-scroller.js 362:20-26
"export 'Easing' was not found in '@tweenjs/tween.js'
ERROR in ./node_modules/ngx-virtual-scroller/__ivy_ngcc__/fesm5/ngx-virtual-scroller.js 360:27-32
"export 'Tween' was not found in '@tweenjs/tween.js'
```

**Comment by cesco69** [comment]
for me raise this error

```
ERROR in Maximum call stack size exceeded

    ERROR in node_modules/ngx-virtual-scroller/virtual-scroller.d.ts:1:23 - erro
r TS2688: Cannot find type definition file for 'tween.js'.

    1 /// <reference types="tween.js" />
```

**Comment by wartab** [comment]
Got the same error.

**Comment by speige** [comment]
Should be fixed in version 4.0.2
Please re-open if you're still having issues.

Sorry about that :(

**Comment by whyboris** [comment]
Confirming issue is no longer present in `4.0.2` 🎉 -- my app compiles and runs 😁 
https://www.npmjs.com/package/ngx-virtual-scroller
Thank you for the great (and quick 🏅) work 🙇
